tests/integration/models/release.spec.js

```diff
@@ -276,6 +276,44 @@ describe('Release Model', function () {
 					});
 				});
 			});
+
+			describe('balena.model.release.setIsInvalidated()', function () {
+				releaseRetrievalFields.forEach((field) => {
+					parallel('', function () {
```

Comment:
This is causing a race condition between invalidating & validating the same release.
Either drop the parallel and let them run in order or check my earlier suggestion which parallelizes the independent test cases.